force-app/main/default/classes/BotController.cls

```diff
@@ -1,4 +1,4 @@
-public with sharing class BotController {
+public class BotController {
```

Comment:
Class `BotController` does not enforce sharing correctly. Please change to `Inherited Sharing`.